packages/pointing-frame/src/LionPointingFrame.js

```diff
@@ -0,0 +1,280 @@
+import { html } from 'lit-html/lit-html.js';
+import { LionLitElement, css } from '../core/LionLitElement.js';
+import { SlotMixin } from '../core/SlotMixin.js';
+import { updatePosition } from '../overlays/utils/manage-position.js';
+import { getPlacement } from '../overlays/utils/get-position.js';
+import pointerSvg from '../../assets/icons/pointer.svg.js';
+
+import '../icon/lion-icon.js';
+
+export class LionPointingFrame extends SlotMixin(LionLitElement) {
+  static get styles() {
+    return css`
+      :host {
+        box-sizing: border-box;
+      }
+    `;
+  }
+
+  get slots() {
+    return {
+      ...super.slots,
+      // TODO: Refactor to put it on instance instead of constructor. No need for clone ;)
```

Comment:
Coming to think of this... there actually is no need at all here for the SlotMixin :)
The general mindset should still be that we don't mess around with light dom ourselves in our components.

Unfortunately there is no way to create accessible components in shadow dom until the [form participation api](http://hangaroundtheweb.com/2018/06/html-form-participation-api-explained/) and the [AOM](https://github.com/WICG/aom/blob/gh-pages/explainer.md) are a reality.
The SlotMixin was created for accessibility purposes. 

Since the pointer is a purely visual thing there would be no problem to place the default pointer in the shadow dom.
We can just use the platform for this in our template:
```html
<slot name="pointer">
    <my-pointer-svg></my-pointer-svg>
</slot>
```
`<my-pointer-svg>` will be visible by default, unless the consuming developer defines the slot himself.

Reply:
Also, I think the `<lion-icon>` is not needed, since it's main purpose is accessibility and a webcomponent for just an svg feels a bit heavier than needed.